From 2b13f9a50793fdc15199925000107cb0cc4e7bad Mon Sep 17 00:00:00 2001
From: Angelo Compagnucci <angelo@amarulasolutions.com>
Date: Thu, 9 Aug 2018 09:28:06 +0200
Subject: [PATCH] FIX: Enabling compiling ppc64le

This patch enables Mender compilation on ppc64le

Changelog: FIX: Enabling compiling ppc64le

Signed-off-by: Angelo Compagnucci <angelo@amarulasolutions.com>
Signed-off-by: Giulio Benetti <giulio.benetti@micronovasrl.com>
---
 ioctl_64_bit.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ioctl_64_bit.go b/ioctl_64_bit.go
index 6f61a6c..f1e2dd8 100644
--- a/ioctl_64_bit.go
+++ b/ioctl_64_bit.go
@@ -1,4 +1,4 @@
-// Copyright 2017 Northern.tech AS
+// Copyright 2018 Northern.tech AS
 //
 //    Licensed under the Apache License, Version 2.0 (the "License");
 //    you may not use this file except in compliance with the License.
@@ -12,7 +12,7 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
-// +build amd64 arm64
+// +build amd64 arm64 ppc64le
 
 package main
 
-- 
2.17.1

